### ISSUE TITLE
Low Priority - WCAG Level AAA - Section headings - text which could be marked up as a heading

### ISSUE THREAD

**Issue by JuliaSang** [w3c/w3c-website#168]
There is text which introduces a section of the page, which would benefit from being marked up as a heading to make it easier for screen reader users to navigate the content.

WCAG Reference:
[Success Criterion 2.4.10 Section Headings (Level AAA) Understanding Section Headings](https://www.w3.org/WAI/WCAG21/Understanding/section-headings.html) | [How to Meet Section Headings](https://www.w3.org/WAI/WCAG21/quickref/#section-headings)

**Issue ID: DAC_Section_headings_01**

URL: https://beta.w3.org/membership/list/ 
Page title: Current Members & Testimonials 
Journey 2, Step 1
<img width="773" alt="heading" src="https://user-images.githubusercontent.com/20167528/159256833-eaaaa88e-2dcf-4ac6-87ae-5dca0b1ddcfe.png">

There is text on the page 'There are 460 members' which visually introduces content and a section of the page. This is not marked-up as a heading, so screen reader users will not be presented the purpose of this information as introductory.

Approximately 70% of screen reader users utilise headings as a means of reviewing the content held on a page; where section headings are not provided this prevents screen reader users from being able to quickly identify the page content using the headings structure.

When a search is initiated by selecting the 'Show results' button, this text updates to reflect the number of results filtered; however this information is not announced to screen reader users. This is because the aria attributes currently in use are only effective when content updates on a page, but not when a new page loads. As performing a search causes the page to be reloaded, this information will not announce.

**Current code ref(s):** #main > div:nth-child(2)
```
<div aria-live="polite" aria-atomic="true">There are 460 members.</div>
```

**Solution:**
Ensure that semantic mark-up is used where appropriate. So that its purpose as introducing content is clear to screen reader users the text 'There are 460 members' should be marked up as a heading at level 2; this will also enable users also to determine quickly from the heading markup how many members are being displayed following submission of a filter search.

Remove the aria attributes that are currently provided as these are intended for use when content dynamically updates on the page. However, as users have to explicitly submit a search to filter results, providing the heading that contains the number of results shown with a role="alert"; this will ensure that this information is announced on page load.

Additionally, so that it is clear that a search has been performed successfully, the information 'Showing search results [number of number]...' should be added to the page title.

**Comment by deniak** [comment]
It's now fixed. Note that the title became too long after adding the result numbers to we didn't change that
